kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/WorkflowNode.kt

```diff
@@ -157,6 +129,26 @@ internal class WorkflowNode<PropsT, StateT, OutputT : Any, RenderingT>(
     )
   }
 
+  override fun <T> runningWorker(
+    worker: Worker<T>,
+    key: String,
+    handler: (T) -> WorkflowAction<StateT, OutputT>
+  ) {
+    // Prevent duplicate workflows with the same key.
+    workers.staging.forEach {
+      require(!(it.matches(worker, key))) {
+        "Expected keys to be unique for $worker: key=$key"
```

Comment:
Helpful to include the toString of both the new worker and the conflicting one in such a message.

Reply:
It's there: `$worker`